test/evergreen/run_format_configs.sh

```diff
@@ -21,20 +20,103 @@ fi
 
 success=0
 failure=0
+running=0
+parallel_jobs=8
+cleanup=0
+PID=""
+declare -a PID_LIST
+
+usage() {
+	echo "usage: $0 "
+	echo "    -j parallel  jobs to execute in parallel (defaults to 8)"
+
+	exit 1
+}
+
+
+while :; do
+	case "$1" in
+	-j)
+		parallel_jobs="$2"
+		[[ "$parallel_jobs" =~ ^[1-9][0-9]*$ ]] ||
+			fatal_msg "-j option argument must be a non-zero integer"
+		shift ; shift ;;
+	-*)
+		usage ;;
+	*)
+		break ;;
+	esac
+done
+
+# Wait for other format runs.
+wait_for_process()
+{
+	while true
+	do
```

Comment:
Suggest making the style consistent with the above line 37.

Reply:
Fixed

Reply:
There's still a `while :;do` in the rework. Can you please double-check?